js/main.js

```diff
@@ -202,14 +194,14 @@ function konamicheck(k)
 $(window).konami({
   cheat: function() {
     isKonaming = !isKonaming;
-
-    $('#menubutton').toggleClass('fa-spin');
-    $('#bgvid').toggleClass('fa-spin');
-    $('#getnewvideo').toggleClass('fa-spin');
-    $('#autonext').toggleClass('fa-spin');
-    $('#skip-left').toggleClass('fa-spin');
-    $('#skip-right').toggleClass('fa-spin');
-    $('#pause-button').toggleClass('fa-spin');
+    
+    document.getElementById("menubutton").toggleClass('fa-spin');
+    document.getElementById("bgvid").toggleClass('fa-spin');
+    document.getElementById("getnewvideo").toggleClass('fa-spin');
+    document.getElementById("autonext").toggleClass('fa-spin');
+    document.getElementById("skip-left").toggleClass('fa-spin');
+    document.getElementById("skip-right").toggleClass('fa-spin');
+    document.getElementById("pause-button").toggleClass('fa-spin');

```

Comment:
just one, big question:

# WHY

Also, in vanilla javascript there's no .toggleClass - http://stackoverflow.com/q/18880890